library/ssl_misc.h

```diff
@@ -770,12 +770,12 @@ struct mbedtls_ssl_handshake_params {
 
 #if (defined(PSA_WANT_ALG_ECDH) || defined(PSA_WANT_ALG_FFDH)) && \
     (defined(MBEDTLS_USE_PSA_CRYPTO) || defined(MBEDTLS_SSL_PROTO_TLS1_3))
-    psa_key_type_t ecdh_psa_type;
-    size_t ecdh_bits;
-    mbedtls_svc_key_id_t ecdh_psa_privkey;
-    uint8_t ecdh_psa_privkey_is_external;
-    unsigned char ecdh_psa_peerkey[DH_PSA_PEERKEY_SIZE];
-    size_t ecdh_psa_peerkey_len;
+    psa_key_type_t dh_psa_type;
```

Comment:
I agree that `dh` makes sense, as the common part between ecdh and ffdh, but I'm slightly worried that it might be a bit ambiguous and might be taken as meaning "ffdh" - for example the PSA key type for `ALG_FFDH` is `KEY_TYPE_DH`.

So, I'm wondering if we should use something like `xxdh` instead in order to be more explicitly.

OTOH, perhaps I'm overthinking this. @mprse @ronald-cron-arm wdyt?

Reply:
`xxdh` is a seems to me a good compromise.